src/Model/Join.php

```diff
@@ -238,7 +238,7 @@ protected function init(): void
             }
 
             if (!$this->foreign_field) {
-                $this->foreign_field = $this->getModelTableString($this->getOwner()) . '_' . $id_field;
+                $this->foreign_field = preg_replace('~^.+?\.~', '', $this->getModelTableString($this->getOwner())) . '_' . $id_field;
```

Comment:
Needs the same change on line 357 of this file

Reply:
yes! - fixed, I think the seeding on l357 was just a duplicated/unneeded code, so I removed it

Reply:
> I couldn't find that button -- is it in githubdesktop, maybe?

available only in "Files changed" tab and near the changed lines only